### ISSUE TITLE
Client does not get connected to the server.

### ISSUE THREAD

**Issue by sree93** [wavetermdev/waveterm#196]
Client does not get connected to the server.

`/lib/x86_64-linux-gnu/libc.so.6: version 'GLIBC_2.34'`

This is basically a regression of this [issue](https://github.com/wavetermdev/waveterm/issues/89#issue-2010687503) 

#89

**Comment by esimkowitz** [comment]
Hi @sree93, I see you changed your title for the issue, can you confirm what version of Wave you are running? The latest release is v0.5.3, released last week. Could you provide some more details on the errors you're seeing? Does anything appear in the `~/.waveterm/wavesrv.log` file that seems notable?

**Comment by sawka** [comment]
You're right this is a regression.  I just checked my x86 linux build and indeed it is dynamically linked.  It was my fault, I used the wrong build method for the linux build (mac still requires dynamic linking).

I just rebuilt the v0.5.3 release (with static linking) and replaced the zip files on the download page. (here's the direct link -- https://dl.waveterm.dev/builds/waveterm-linux-x64-v0.5.3.zip ).

thanks for submitting the issue!
